display/ssd1306.h

```diff
@@ -567,6 +619,8 @@ class SSD1306 : public I2CDevice, Looper, StateMachine, SaberBase, private Audio
   uint16_t i;
   uint8_t xor_ = 0;
   bool invert_y_ = 0;
+  bool looped_on_ = false;
+  Effect* current_effect_;
```

Comment:
I think this needs to be volatile, because it's used in both interrupts and non-interrupts.

Reply:
bool looped_on_ marked as volatile